app/__init__.py

```diff
@@ -39,20 +44,23 @@ def create_app(config_class=Config):
         ]
     )
 
+    app.jinja_env.filters["null_to_dash"] = null_to_dash
+
     # Set content security policy
     csp = {
         "default-src": "'self'",
         "script-src": ["'self'"],
     }
 
-    app.jinja_env.filters["null_to_dash"] = null_to_dash
+    # Disable https if app is run in testing mode
+    force_https = True if app.config["TESTING"] else False
```

Comment:
took a while to figure out what was causing redirects in the test flask client requests - turns out it is talisman. This turns it off in tests: https://github.com/GoogleCloudPlatform/flask-talisman/issues/50.

If we want to test that at all we can override on particular tests or more than likely test it in our e2e test suite.